Description: Replace RedHat style font path with Debian style
 RedHat has a different layout for fonts than Debian.
 .
 nextspace-frameworks (0.85+20200526.gitfc5851fd-1) unstable; urgency=medium
 .
   * Update to latest master
Author: Patrick Georgi <patrick@georgi.software>

---
Last-Update: 2020-06-02

--- nextspace-frameworks-0.85+20200526.gitfc5851fd.orig/DesktopKit/Resources/Fonts/RobotoMono.nfont/FontInfo.plist
+++ nextspace-frameworks-0.85+20200526.gitfc5851fd/DesktopKit/Resources/Fonts/RobotoMono.nfont/FontInfo.plist
@@ -4,7 +4,7 @@
     Faces = (
 	{
 	    Files = (
-		/usr/share/fonts/google-roboto-mono/RobotoMono-Regular.ttf
+		/usr/share/fonts/truetype/roboto-mono/RobotoMono-Regular.ttf
 	    );
 	    Name = Regular;
 	    PostScriptName = "RobotoMono-Regular";
@@ -13,7 +13,7 @@
 	},
 	{
 	    Files = (
-		/usr/share/fonts/google-roboto-mono/RobotoMono-Bold.ttf
+		/usr/share/fonts/truetype/roboto-mono/RobotoMono-Bold.ttf
 	    );
 	    Name = Bold;
 	    PostScriptName = "RobotoMono-Bold";
@@ -22,7 +22,7 @@
 	},
 	{
 	    Files = (
-		/usr/share/fonts/google-roboto-mono/RobotoMono-Italic.ttf
+		/usr/share/fonts/truetype/roboto-mono/RobotoMono-Italic.ttf
 	    );
 	    Name = Italic;
 	    PostScriptName = "RobotoMono-Italic";
@@ -31,7 +31,7 @@
 	},
 	{
 	    Files = (
-		/usr/share/fonts/google-roboto-mono/RobotoMono-BoldItalic.ttf
+		/usr/share/fonts/truetype/roboto-mono/RobotoMono-BoldItalic.ttf
 	    );
 	    Name = BoldItalic;
 	    PostScriptName = "RobotoMono-BoldItalic";
@@ -40,7 +40,7 @@
 	},
 	{
 	    Files = (
-		/usr/share/fonts/google-roboto-mono/RobotoMono-Medium.ttf
+		/usr/share/fonts/truetype/roboto-mono/RobotoMono-Medium.ttf
 	    );
 	    Name = Medium;
 	    PostScriptName = "RobotoMono-Medium";
@@ -49,7 +49,7 @@
 	},
 	{
 	    Files = (
-		/usr/share/fonts/google-roboto-mono/RobotoMono-MediumItalic.ttf
+		/usr/share/fonts/truetype/roboto-mono/RobotoMono-MediumItalic.ttf
 	    );
 	    Name = MediumItalic;
 	    PostScriptName = "RobotoMono-MediumItalic";
